examples/deckgl-mosaic-datafusion/README.md

```
@@ -0,0 +1,46 @@
+### Mosaic crossfilter with DataFusion-WASM and Zarr, rendered with deck.gl
+
+This example ports [Gjore Milevski](https://github.com/dzole0311)'s
+[mosaic-datafusion-zarr-deckgl](https://github.com/dzole0311/mosaic-datafusion-zarr-deckgl)
+experiment (write-up [here](https://gjoremilevski.com/posts/mosaic-datafusion-zarr-deckgl/))
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)